src/MenuItem.jsx

```diff
@@ -11,37 +11,37 @@ function MenuItem() {
         <ListItem>
           <Link
             component={RouterLink}
-            to="/employees"
+            to="/admin"
             sx={{ textDecoration: "none" }}
           >
-            Employee Management
+            Admin
           </Link>
-        </ListItem>
+        </ListItem>{" "}
```

Comment:
Redundant comment

Reply:
It's the IDE that added it... lol 
Fixed!!